dist/codecov_ats.sh

```diff
@@ -133,6 +133,9 @@ if [[ -z $ats_tests_to_run ]]; then
     else
         ats_tests_to_run=${ats_tests_to_skip_array[ $RANDOM % ${#ats_tests_to_skip_array[@]} ]}
     fi
+elif [[ -z $ats_tests_to_skip ]]; then
+    say "$y==>$x No tests are skipped, running all"
+    ats_tests_to_run=""
```

Comment:
Ideally we'd still include `--cov-context=test` in here.

Reply:
@giovanni-guidini yup, this is appended to that

```
test_commands="$runner_options "
test_commands+=$ats_tests_to_run
```

Reply:
ah perfect then